<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UltraPhoto2000</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionNouvel_album"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEditition">
    <property name="title">
     <string>É&amp;dition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionR_tablir"/>
    <addaction name="separator"/>
    <addaction name="actionPr_f_rences"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="actionAide_en_ligne"/>
    <addaction name="actionA_propos"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Aff&amp;ichage</string>
    </property>
    <addaction name="actionPlein_cran"/>
    <addaction name="actionRecharger"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEditition"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAide_en_ligne">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/ressources/images/HELP-01.png</normaloff>:/ressources/images/HELP-01.png</iconset>
   </property>
   <property name="text">
    <string>Aide</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/ressources/images/INFO-01.png</normaloff>:/ressources/images/INFO-01.png</iconset>
   </property>
   <property name="text">
    <string>À propos</string>
   </property>
  </action>
  <action name="actionPr_f_rences">
   <property name="text">
    <string>Préférences</string>
   </property>
  </action>
  <action name="actionNouvel_album">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/ressources/images/NEWALBUM-01.png</normaloff>:/ressources/images/NEWALBUM-01.png</iconset>
   </property>
   <property name="text">
    <string>Nouvel album...</string>
   </property>
  </action>
  <action name="actionRecharger">
   <property name="text">
    <string>Recharger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPlein_cran">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plein écran</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionR_tablir">
   <property name="text">
    <string>Rétablir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom+</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom-</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
